libs/payments/stripe/src/lib/stripe.manager.ts

```diff
@@ -74,4 +76,36 @@ export class StripeManager {
       customer.tax?.automatic_tax === 'not_collecting'
     );
   }
+
+  /**
+   * If customer taxId does not match incoming taxId, update customer record with incoming taxId.
+   *
+   * @param customerId
+   * @param taxId
+   * @returns
+   */
+  async addTaxIdToCustomer(customerId: string, taxId: string) {
```

Comment:
Thinking about this in hindsight after writing the ticket - Probably best to rename this to `setCustomerTaxId` since a customer can only have only one tax id assigned.

Reply:
Incorrect - ignore.